From 50c0f13f99454d292d3fe8315d9f0a7a4b5a0e72 Mon Sep 17 00:00:00 2001
From: juztamau5 <juztamau5@gmail.com>
Date: Sun, 12 Jun 2022 00:02:09 -0700
Subject: [PATCH 1/3] Fix compiler error

Error was:

  RefundEscrow.sol:83:9: TypeError: Member "sendValue" not found or not visible after argument-dependent lookup in address payable.
          beneficiary().sendValue(address(this).balance);
          ^---------------------^
---
 contracts/payment/escrow/RefundEscrow.sol | 3 +++
 1 file changed, 3 insertions(+)

diff --git a/contracts/payment/escrow/RefundEscrow.sol b/contracts/payment/escrow/RefundEscrow.sol
index 21a6e8df..12f88451 100644
--- a/contracts/payment/escrow/RefundEscrow.sol
+++ b/contracts/payment/escrow/RefundEscrow.sol
@@ -3,6 +3,7 @@
 pragma solidity >=0.6.0 <0.8.0;
 
 import "./ConditionalEscrow.sol";
+import "../../utils/Address.sol";
 
 /**
  * @title RefundEscrow
@@ -15,6 +16,8 @@ import "./ConditionalEscrow.sol";
  * with `RefundEscrow` will be made through the owner contract.
  */
 contract RefundEscrow is ConditionalEscrow {
+    using Address for address payable;
+
     enum State { Active, Refunding, Closed }
 
     event RefundsClosed();
-- 
2.34.1

